pkg/docker-py3/build.py

```diff
@@ -0,0 +1,14 @@
+import docker
+import gramex.cache
+import io
+
+template = gramex.cache.open('Dockerfile.tmpl', 'template', rel=True)
+release = gramex.cache.open('../../gramex/release.json', rel=True)
+client = docker.from_env()
+streamer = client.api.build(
+    fileobj=io.BytesIO(template.generate(**release)),
+    tag=f'gramener/gramex:{release["version"]}',
+    decode=True,
+)
```

Comment:
Please add `rm=True` as an argument here, otherwise daemon creates a lot of intermediate images and doesn't clean them up on a successful build.